src/screens/user-management/types.ts

```diff
@@ -0,0 +1,47 @@
+import type { ChangeEvent, SyntheticEvent } from "react";
+import type { BaseQueryParams } from "@/types/query-params";
+import type { OrderType } from "@/types/sort";
+
+export interface FilterProps {
+  handleChangeInput: (
+    e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => void;
+  handleChangeSelectStatus: (_: SyntheticEvent, options: any) => void;
+  handleChangeSelectPagination: (_: SyntheticEvent, options: any) => void;
```

Comment:
options is any because MuiAutocomplete type for ```value``` is any too 🥲

![Screenshot 2567-09-25 at 10 23 57](https://github.com/user-attachments/assets/1711459b-5cd6-4437-8a31-fd9fc7c4f288)